resources/xml/fedramp_values.xml

```diff
@@ -495,7 +495,7 @@
          <enum value="procedure" short-label="Procedure">Procedure</enum>
          <enum value="guide" short-label="Guidance">Guidance Document</enum>
          <enum value="pia" short-label="P.I.A.">Privacy Impact Assessment</enum>
-         <enum value="rob" short-label="R.O.B.">Rules of Behavior</enum>
+         <enum value="rules-of-behavior" short-label="R.O.B.">Rules of Behavior</enum>
```

Comment:
should we expand all the abbreviations?

Reply:
I think they should all be subjected to some adult supervision. They look like they were arbitrary choices by a single individual.